pkgs/development/node-packages/node-env.nix

```diff
@@ -547,8 +551,8 @@ let
             python
             nodejs
           ]
-          ++ lib.optional (stdenv.hostPlatform.isLinux) pkgs.util-linux
-          ++ lib.optional (stdenv.hostPlatform.isDarwin) libtool
+          ++ lib.optional (stdenv.isLinux) utillinux
```

Comment:
Please restore `stdenv.hostPlatform` here

Reply:
Done. This change is the result of running the node packages update command. Maybe we should have a look at why it does this.